bubble_sort/Python/bubble_sort.py

```diff
@@ -0,0 +1,14 @@
+def bubbleSort(arr): #Function that performs the bubbleSort
+
+  n = len(arr) #Finding the length of the array using the len function 
+
+  for i in range(n): # Traverse through all array elements 
+    for j in range(0, n-i-1): # Traverse the array from 0 to n-i-1 
+        if arr[j] > arr[j+1]: 
+            arr[j], arr[j+1] = arr[j+1], arr[j] # Swap if the element found is greater than the next element 
+              
+arr = [12, 96, 24, 48, 36, 60, 84, 72] # Declaring a sample array
+bubbleSort(arr) # Calling the bubbleSort function and passing the array 
+print ("Sorted array is:") # Printing the array after sorting it.
+for i in range(len(arr)): 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
@@ -10,5 +10,5 @@
 arr = [12, 96, 24, 48, 36, 60, 84, 72] # Declaring a sample array
 bubbleSort(arr) # Calling the bubbleSort function and passing the array 
 print ("Sorted array is:") # Printing the array after sorting it.
-for i in range(len(arr)): 
+for i in range(len(arr)):
   print(arr[i])
```